packages/dart_frog/lib/src/serve.dart

```diff
@@ -8,11 +8,14 @@ part of '_internal.dart';
 /// By default, the header will be:
 ///
 /// `"X-Powered-By": "Dart with package:dart_frog"`
+///
+///  If a [securityContext] is provided an HTTPS server will be started
```

Comment:
Aside from dart docs, we should also have an entry about this in the advanced section about using your own `run` method.

Reply:
Added documentation file